lib/valid_email2/address.rb

```diff
@@ -85,12 +88,16 @@ def disposable_mx_server?
     end
 
     def whitelisted?
-      domain_is_in?(ValidEmail2.whitelist)
+      domain_is_in?(ValidEmail2.allow_list)
     end
+    alias_method :allow_listed?, :whitelisted?
+    deprecate_method :whitelisted?, :allow_listed?
```

Comment:
```suggestion
    alias_method :whitelisted?, :allow_listed?
    deprecate_method :whitelisted?, :allow_listed?
```
the general pattern is to rename the methods to the new target and alias the previous method names

Reply:
Got it! Thanks for the tip 🙇